build/Dockerfile.base.arm64

```diff
@@ -1,7 +1,23 @@
-FROM --platform=linux/arm64 registry.access.redhat.com/ubi8/ubi:8.4
+FROM docker.io/nvidia/cuda:11.8.0-base-ubi8
 
-ARG ARCH=arm64
+ARG ARCH=aarch64
```

Comment:
looks like we do support ARM, but kepler doc seems not talk about it before ..
good to know